app/scripts/controllers/rif/rns/resolver/index.js

```diff
@@ -186,12 +193,19 @@ export default class RnsResolver extends RnsJsDelegate {
    * @param domainName DomainName with the .rsk extension
    * @param chain
    * @param chainAddress
+   * @param subdomain
    * @returns {Promise<unknown>}
    */
-  setChainAddressForResolver (domainName, chain, chainAddress) {
+  setChainAddressForResolver (domainName, chain, chainAddress, subdomain = '') {
     return new Promise((resolve, reject) => {
+      let node = namehash.hash(domainName);
+      if (subdomain && subdomain !== '') {
```

Comment:
The same here i don't know if this is necessary, when you check if subdomain !== '' it's the same as asking if (subdomain) since the if checks for undefined, null and empty strings so i think that is not necessary.